### DISCUSSION TITLE
[Bug] LinkedIn URLs

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#3081] [Support]
Thank you for adding the extra social links and moving them to the top of the home page. 

One issue, though: our LinkedIn is a company page, so the auto-populated `linkedin.com/in/` doesn't work because companies are under `linkedin.com/company/`.

Like so: https://www.linkedin.com/company/san-antonio-review

Not sure how best to approach fixing this; not sure there's a one-size-fits-all solution.

**Comment by eastofesten** [comment]
Thanks for highlighting this! I created an [issue](https://github.com/pubpub/pubpub/issues/3082) from this discussion for us to look into.
